### ISSUE TITLE
Remove use of deprecated property in `setCursor` in Google Adapter

### ISSUE THREAD

**Issue by AndrewADev** [JamesLMilner/terra-draw#104]
Currently, the Google Maps adapter is relying on a [deprecated property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLStyleElement/type) while setting styles for the cursor:
![image](https://github.com/JamesLMilner/terra-draw/assets/2937540/981a05cc-7a0c-4570-b566-f24951781922)

This should probably be removed.

This may also be a good time to revisit how styles are being constructed for the Google Maps adapter: Not sure if there's a reason why this adapter directly constructs a style element in the head of the document/globally?

If not, we could try setting styles on the container `div` directly, such as is [done for the arcGIS adapter](https://github.com/JamesLMilner/terra-draw/blob/main/src/adapters/arcgis-maps-sdk.adapter.ts#L119).

Otherwise, if it needs to be global for some reason, we could target the document in a slightly more generic way, via [constructable stylesheets](https://web.dev/articles/constructable-stylesheets?hl=en) (something [like this sandbox demo](https://codesandbox.io/s/constructable-style-sheets-pl6gw6?file=/src/index.mjs:561-663)) :

**Comment by JamesLMilner** [comment]
Thanks for raising this!

As you say I think we can remove it as it is deprecated and redundant as the default value is `text/css`. 

So if I remember correctly this was done because there didn't seem a good consistent API that for setting the cursor that worked in the way I expected. There maybe a less intrusive or more performant way to do this than manually inserting a new style sheet each time the cursor changes so definitely open to exploring that as long as it's well supported across browsers.

**Comment by AndrewADev** [comment]
It looks like there is also a bug here - the styles won't work when the requesting language is different than English (for any language where it is not captioned as a 'map'), as the selector [currently leverages the `aria-label`](https://github.com/JamesLMilner/terra-draw/blob/main/src/adapters/google-maps.adapter.ts#L164) attribute, which is localized based on the requested language:
![image](https://github.com/JamesLMilner/terra-draw/assets/2937540/81c9e1c3-a591-40f5-91c1-e96fefa2b27c)


Note that I didn't explicitly change anything in the setup - the SDK seems to be determining this from my browser settings (Chrome in German):

![image](https://github.com/JamesLMilner/terra-draw/assets/2937540/4f43b741-687b-40f1-8341-673ba1a1d475)


If I explicitly set the requesting language to English, like so:
```typescript
	const loader = new Loader({
		apiKey,
		version: "weekly",
		language: "en",
	});
```

...then I see the cursor style I expect  - tested in `TerraDrawPointMode` in a modified version of the development space:

```typescript
		new TerraDrawPointMode({
			cursors: {
				create: "crosshair",
			}
		}),
```

With that said, not sure if its better to update this issue or open another one for better change tracking? Same goes for the PR(s) to address the issues.

**Comment by JamesLMilner** [comment]
Hey @AndrewADev thanks for pointing this out - this indeed a bug. I have raised a separate issue and a PR to fix this, it's not pretty but should resolve the issue for now. I think we should create a separate issue for looking at handling the styling from `setCursor` in the Google Maps adapter more gracefully. 

I also pushed a fix for the removal of setting the type to `text/css` so that is now resolved and this can be closed!
